cmd/minikube/cmd/start.go

```diff
@@ -1030,8 +1030,18 @@ func validateNetwork(h *host.Host, r command.Runner) string {
 
 func trySSH(h *host.Host, ip string) {
 	sshAddr := fmt.Sprintf("%s:22", ip)
-	conn, err := net.Dial("tcp", sshAddr)
-	if err != nil {
+
+	dial := func() (err error) {
+		dialer := net.Dialer{Timeout: 3 * time.Second}
+		conn, err := dialer.Dial("tcp", sshAddr)
+		if err != nil {
+			return err
+		}
+		_ = conn.Close()
+		return nil
+	}
+
+	if err := retry.Expo(dial, time.Second, 10*time.Second); err != nil {
```

Comment:
on a second thought, lets increase this to 13, since 10 seems to be the cutting time. give it a with of wiggle room.

lets change time.Second to 300 milliseconds.

Reply:
I personally feel 250ms is really aggressive for an issue that should never happen.

Reply:
I am okay with 1 second, but the timeout time needs to be increased from 10

Reply:
Changed timeout to 13 seconds, keeping the interval at 1 second.